DependencyInjection/CmfRoutingExtension.php

```diff
@@ -371,14 +384,32 @@ private function loadConditionalEnhancer(array $config, ContainerBuilder $contai
         $container->addDefinitions($definitions);
     }
 
-    private function buildConditionalEnhancerDefintion($className, $source, $target, $map, $methods, $priority)
+    private function buildConditionalEnhancerDefintion($className, $source, $target, $maps, $priority)
     {
+        var_dump($maps);throw new \Exception;
+        if (1 === count($maps) && !array_key_exists('methods', reset($maps))) {
```

Comment:
my idea is to avoid extra wrappers when they are not needed. but i guess you are right that methods=any should also be this case.